landing.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="landing.css">
+        <script src="https://kit.fontawesome.com/7d2d9d3834.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header id="header">
+            <div class="icon-title">
+                <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" id="header-img">
+                <!-- <h2 class="header-title">Original Trombones</h2> -->
+            </div>
+            
+            <nav id="nav-bar">
+                <ul class="menu-links">
+                    <li> <a href="features">features</a> </li>
+                    <li> <a href="Ho">How it works</a> </li>
+                    <li> <a href="">Pricing</a> </li>
+                </ul>
+            </nav>
+        </header>
+        <form>
+            <h3> Handcrafted, home-made masterpieces</h3>
+            <input type="email" placeholder="Enter your Email address">
+            
+            <div><button type="submit">GET STARTED</button></div>
+            
+        </form>
+        <div class="banner">
+            
+        <div class="icon-description">
```

Comment:
It will be nice if you use a section instead of a div for different sections of your project.